### DISCUSSION TITLE
使用手机刷而非模拟器

### DISCUSSION THREAD

**Discussion by Kyakui** [MaaAssistantArknights/MaaAssistantArknights#2472] [Ideas]
请问可以添加用电脑连接手机，用手机刷吗

**Comment by zzyyyl** [comment]
参考[模拟器支持 - 手机、平板等非 `16:9` 分辨率设备](https://github.com/MaaAssistantArknights/MaaAssistantArknights/blob/d8c922eb6a1e1ef7a5995c52141d00d6bb57a5c9/docs/1.3-%E6%A8%A1%E6%8B%9F%E5%99%A8%E6%94%AF%E6%8C%81.md#%EF%B8%8F-%E6%89%8B%E6%9C%BA%E5%B9%B3%E6%9D%BF%E7%AD%89%E9%9D%9E-169-%E5%88%86%E8%BE%A8%E7%8E%87%E8%AE%BE%E5%A4%87)

**Comment by issenn** [comment]
@zzyyyl 
[asst.log](https://github.com/MaaAssistantArknights/MaaAssistantArknights/files/10184870/asst.log)
不太行，调整过分辨率也识别不到，开基建功能的话提示是截图失败。
